.clj-kondo/hooks/common.clj

```diff
@@ -200,6 +200,34 @@
                 body))]
     {:node node*}))
 
+(defn- let-second-inner [body bindings]
+  (let [binding-infos (for [[model {[binding value] :children}] (partition 2 bindings)]
+                        {:model   model
+                         :binding binding
+                         :value   (or value
+                                      (hooks/token-node 'nil))})]
+    (-> (hooks/vector-node
+         [(hooks/vector-node (map :model binding-infos))
+          (-> (hooks/list-node (list* (hooks/token-node `let)
+                                    (hooks/vector-node (mapcat (juxt :binding :value) binding-infos))
+                                    body))
+              (with-meta (meta body)))])
+        (with-meta (meta body)))))
+
+(defn let-second
+  "Helper for macros that have a shape like, where the let is for second arg.
```

Comment:
Helper for macros that have a shape like ???

I think you're missing something here, or this sentence needs rewriting. You should also note the first argument appear to be used